extensions/libraries/redcore/api/soap/soap.php

```diff
@@ -211,42 +279,6 @@ public function apiWsdl()
 	}
 
 	/**
-	 * Checks and ensures that a static WSDL file exist and is in place
```

Comment:
It's to help me read through the code. I find it easier to keep track of what is going on if the methods used in the method I'm using are arranged in order of execution below the method.